woocommerce/woocommerce-gutenberg-products-block
# Remove useCheckoutRedirectUrl()

I can't find any usage of this hook. Everything seems to work fine after removing it (tested bank transfer, Stripe + 3DS, saved cards and express payment methods).

### How to test the changes in this Pull Request:

Smoke test the checkout flow with different payment methods and verify everything keeps working as usual.
